htdocs/product/list.php

```diff
@@ -1665,6 +1682,23 @@
 		}
 	}
 
+	// Default Workstation
+	if (!empty($arrayfields['p.fk_default_worksation']['checked'])) {
+		print '<td align="center">';
+		if (!empty($obj->fk_default_workstation)) {
+			$static_ws = new Workstation($db);
```

Comment:
Move the new out of the loop, at begin of file, and reuse same object on each loop pass.